dom0-updates/qubes-dom0-update

```diff
@@ -166,6 +166,8 @@ rm -f /var/lib/qubes/updates/errors
 echo "Using $UPDATEVM as UpdateVM to download updates for Dom0; this may take some time..." >&2
 
 # qvm-run by default auto-starts the VM if not running
+qvm-run --nogui -q -u root  $UPDATEVM 'mkdir -m 775 /var/lib/qubes/dom0-updates/' || exit 1
```

Comment:
This on the other hand will fail if the directory exists. Add `-p`.

Reply:
Ok Marek, at least in Qubes 3.2 there was no problem since qvm-run ignores exit code without --pass-io, but it seems better this way.